dpnp/backend/queue_sycl.cpp

```diff
@@ -118,7 +119,39 @@ void backend_sycl::backend_sycl_queue_init(QueueOptions selector)
     std::cout << "SYCL kernels link time: " << time_kernels_link.count() << " (sec.)\n" << std::endl;
 }
 
+void backend_sycl::backend_sycl_rng_engine_init()
+{
+    if (rng_engine)
+    {
+        backend_sycl::destroy_rng_engine();
+    }
+    // TODO:
+    // choose engine as is in numpy
+    // seed number
+    size_t seed = std::time(nullptr);
+    rng_engine = new mkl_rng::philox4x32x10(DPNP_QUEUE, seed);
+}
+
+void backend_sycl::backend_sycl_rng_engine_init(size_t seed)
```

Comment:
Use default argument for the function to avoid function duplication
https://en.cppreference.com/w/cpp/language/default_arguments